01_temperature/temperature.rb

```diff
@@ -1 +1,8 @@
-#write your code here
+
+def ftoc(ftoc)
```

Comment:
Function works great. I will just mention that the naming convention for this would be something like this: fahrenheit_to_celsius(number) or f_to_c.